ntcore/src/main/native/cpp/net/WebSocketConnection.cpp

```diff
@@ -226,6 +226,11 @@ int WebSocketConnection::Flush() {
   m_ws_frames.clear();
   if (m_err) {
     m_frames.clear();
+    for(auto& _buf : m_bufs) {
+      if(_buf.len == 0) {
+        _buf.Deallocate();
+      }
+    }
     m_bufs.clear();
```

Comment:
What does m_bufs.clear() do if already deallocated

Reply:
From what I understand it has nothing to do with the deallocation since the `Buffer` class does not have a destructor. Previously, this meant that all the references stored in the vector were lost without being properly deallocated.

Reply:
I suspected it would be something along those lines, in that case this lgtm